### PR TITLE
Adding pin to the map :)

### PR DESCRIPTION
**Pull request by tonuvasilemarius**
Adding me to the map.

### CODE REVIEW

**Comment by rchillard** [comment]
Well done, keep going!

**Comment by shajanjp** [comment]
Looks great ! Now press the big green **Merge pull request** button below and your coordinates will be added to the map. @tonuvasilemarius

**Comment by NikitoZZZZZ** [comment]
Great work @tonuvasilemarius. Looks good to me.

File: _pins/Marius-London.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: Marius-London
```
Comment (hectorsector) [inline _pins/Marius-London.json]: @tonuvasilemarius looks like you're ready to merge -- you may want to change this to your username though!

**Pull request merged**